jdbc/src/java/org/apache/hive/jdbc/Utils.java

```diff
@@ -142,6 +142,8 @@
     static final String WM_POOL = "wmPool";
     // Cookie prefix
     static final String HTTP_COOKIE_PREFIX = "http.cookie.";
+    // Create external purge table by default
+    static final String DEFAULT_EXTERNAL_TABLE = "defaultExternalTable";
```

Comment:
I think this will need to be documented in the wiki

Reply:
There is requirement change for the jira, I am working on another patch. Will document once the code is merged.